requirements-py-docs.txt

```diff
@@ -0,0 +1,4 @@
+Sphinx==1.4.4
```

Comment:
Not yet; I figured we'd wait until this PR is merged (though I did note the need to change it in a comment above).